hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -152,7 +186,7 @@ define([
             },
             'pay': {},
             'hour': {},
-            'leave': [],
+            'leave': ['ses'],
```

Comment:
What is `ses`?

Reply:
"Ses" means "Developer forgot to remove debugging code".